php-api-pro/problem-detail.php

```diff
@@ -33,3 +33,4 @@ public function type(): string
 }
 
 $statusCode = 403;
+var_dump(ProblemDetail::tryFrom($statusCode)->type());
```

Comment:
Almost..one slight problem though. If tryFrom returns null, there will be an error when you try to call type()